### DISCUSSION TITLE
cosmos model background (b) distribution hyperparameters

### DISCUSSION THREAD

**Discussion by gelles-brandeis** [gelles-brandeis/tapqir#424] [Q&A]
If I understand correctly, the cosmos model produces a pair of mu^b and sigma^b values for each AOI.  Is there a way to access these?  They do not seem to be present in the cosmos_params.mat files.  thanks, Jeff

**Comment by ordabayevy** [comment]
Hi Jeff,

They should be in the `background` field. Do you see other fields like `height` or `width`?
